frontend/src/components/story/StoryDay.vue

```diff
@@ -35,6 +35,8 @@
             </h4>
             <api-form :entity="chapter">
               <api-richtext
+                class="e-story-day"
```

Comment:
I guess this something unrelated to the CategoryChip? Or some visual impact between the chip above and this part?

Reply:
Impact for visual hierarchy

Reply:
Hmm, it's very difficult to understand what `e-story-day--textmode` is doing without trying it out. I just tried it out, now I understand 🤣 

Some comment/documentation? Why do we want the difference between read mode & edit mode?

Reply:
Where would you expect a comment/documentation? Because it is more readable.

Reply:
```css
/* this disables the bottom border which is displayed for VTextField in "regular" style */
.e-story-day--textmode :deep(.v-input__slot)::before {
  display: none;
}
```
Sorry, but this was entirely not clear to me without running the code and looking at the difference.